core/src/scss/utilities/variables/core/_magic-numbers.scss

```diff
@@ -1,64 +1,47 @@
 @charset "UTF-8";
 
-//
-// Magic Numbers
-//
-// Unique and resuable values with multiple occurrences.
-//
-// Style guide: Variables.Core.Magic
-//
+///
+/// Magic Numbers
+///
+/// Unique and resuable values with multiple occurrences.
+///
 
-//
-// $su-lead-max-width
-//
-// For setting the max-width for the lead font style.
-//
-// Markup: $su-lead-max-width: 85rem !default;
-//
-// Styleguide: Variables.Core.Magic.su-lead-max-width
-//
+///
+/// For setting the max-width for the lead font style.
+///
+/// @name $su-lead-max-width
+///
+/// @group magic-numbers
 $su-intro-max-width: 85rem !default;
 
-//
-// $su-site-max-width
-//
-// For setting the max-width of the project's grid system.
-//
-// Markup: $su-site-max-width: 1500px !default;
-//
-// Styleguide: Variables.Core.Magic.su-site-max-width
-//
+///
+/// For setting the max-width of the project's grid system.
+///
+/// @name $su-site-max-width
+///
+/// @group magic-numbers
 $su-site-max-width: 1500px !default;
 
-//
-// $su-input-max-width
-//
-// For setting the max-width of input fields.
-//
-// Markup: $su-input-max-width: 46rem !default;
-//
-// Styleguide: Variables.Core.Magic.su-input-max-width
-//
+///
+/// For setting the max-width of input fields.
+///
+/// @name $su-input-max-width
+///
+/// @group magic-numbers
 $su-input-max-width: 46rem !default;
 
-//
-// $su-border-radius
-//
-// For setting the default border-radius.
-//
-// Markup: $su-border-radius: 3px !default;
-//
-// Styleguide: Variables.Core.Magic.su-border-radius
-//
+///
+/// For setting the default border-radius.
+///
+/// @name $su-border-radius
+///
+/// @group magic-numbers
 $su-border-radius: 3px !default;
```

Comment:
Should we get rid of this section? I don't believe we ever use this one.

Reply:
This is used in skiplinks and radio boxes.